src/Theme.elm

```diff
@@ -265,4 +269,130 @@ containerContent children =
 
 page : List (Html msg) -> Html msg
 page children =
```

Comment:
Will maybe move this out in a next task.

Reply:
Yep, I think the function, plus exit button markup/style/function should all be with this and in a pagetemplate file